dev/game.py

```diff
@@ -0,0 +1,249 @@
+class Sprite():
+    def __init__(self, name, width=1, height=1):
+        self.x = -1
+        self.y = -1
+        self.width = width
+        self.height = height
+        self.name = name
+
+class Game():
+    def __init__(self, x,y,debugger=False):
+        self.x = x
+        self.y = y
+        self.grid = []
+        self.ids =["empty"]
+        self.sprites = []
+        self.debugger = debugger
+        #Can't use numpy :-(
+        #Creates grid
+        self.grid = Game._make_grid(self.x,self.y)
+    #Returns what is found at grid location
+
+    #Allows for default game and sprites to be added, for game levels etc
+    def add_preset_grid(grid,ids,sprites=[]):
+        #Doesn't error handle case where more id numbers in grid than in self.ids
+        self.grid = grid
+        self.ids = ids
+        self.sprites = sprites
+    
+    #Updates entire grid with new grid
+    def update_grid(grid):
+        self.grid = grid
+    #Adds new sprite id(string)
+    def add_id(ID):
+        self.ids.append(ID)
+   
+    #Checks what is at a given coOrd
```

Comment:
Is the ` coOrd` spelling intentional? I beleive coord is correct. (Just a minor but recurring thing)